_data/glossary/volume.yaml

```diff
@@ -0,0 +1,18 @@
+id: volume
+name: Volume
+full-link: kubernetes.io/docs/concepts/storage/volumes/
+related:
+- pod
+- container
+- secret
+tags:
+- core-object
+- fundamental
+short-description: >
+ A volume is a directory containing data, which is accessible to the containers in a {% glossary_tooltip text="pod" term_id="pod" %}.
+long-description: >
```

Comment:
Strike "A volume is", and start with "A directory". The plan is to automatically insert the short definition, in the concept topic, after "A volume is".